google-cloud-storage/samples/acceptance/files_test.rb

```diff
@@ -492,6 +492,7 @@ def mock_cipher.random_key
     assert_includes out, "<input name='policy'"
     assert_includes out, "<input name='x-goog-meta-test' value='data'"
     assert_includes out, "<input type='file' name='file'/>"
+    assert_includes out, "<input type='submit' value='Upload File'/>"
```

Comment:
Chris, this wasn't originally added, but would you be open to adding an actual request using the form data into the test?

Reply:
Added testing with Net::HTTP POST, which is not exactly using the "form data". Let me know if you had something else in mind.